### PR TITLE
Jshint fixes

### PR DESCRIPTION
**Pull request by prymitive**
Fix warnings from jshint

### CODE REVIEW

**Comment by bobrik** [comment]
Should jslint be a part of travis build then?

**Comment by prymitive** [comment]
Yes, assuming it can find more than just missing semicolons.
But it needs npm, so maybe only for travis, as I don't want to force people to install npm.

**Comment by prymitive** [comment]
Yup, that's what I did.
`make test` will use jshint if it's installed & travis will now install it

File: .travis.yml [hunk 1]
```diff
@@ -6,5 +6,15 @@ go:
 
 go_import_path: github.com/cloudflare/unsee
 
+# nodejs is needed so that we can install jshint
+addons:
+  apt:
+    packages:
+    - nodejs
```
Comment (bobrik) [inline .travis.yml]: Nit: indent with 2 more spaces for consistency.

**Comment by prymitive** [comment]
Few more updates for the added benefit of this actually working. Pushed bad commit here and got broken build, removed broken commit and got green build.

**Pull request merged**